README.md

```diff
@@ -672,6 +673,29 @@ fs.mkdir('/home', function(err) {
 });
 ```
 
+#### fs.access(path, [mode], callback)<a name="access"></a>
+
+Tests a user's permissions for the file or directory supplied in `path` argument. Asynchronous [access(2)](http://pubs.opengroup.org/onlinepubs/009695399/functions/access.html). Callback gets no additional arguments. The `mode` argument can be:
```

Comment:
"...argument can be one of the following (constants are available on `fs.constants`):"